staging/src/k8s.io/dynamic-resource-allocation/controller/controller_test.go

```diff
@@ -208,16 +225,23 @@ func TestController(t *testing.T) {
 
 		// pod with no claims -> shouldn't occur, check again anyway
 		"pod-nop": {
-			key:                   podKey,
+			key:                   podSchedulingCtxKey,
 			pod:                   pod,
 			schedulingCtx:         withSelectedNode(podSchedulingCtx),
 			expectedSchedulingCtx: withSelectedNode(podSchedulingCtx),
-			expectedError:         errPeriodic.Error(),
+			expectedWorkQueueState: Mock[string]{
+				Later: []MockDelayedItem[string]{
+					{
+						Item:     podSchedulingCtxKey,
+						Duration: time.Second * 30,
```

Comment:
Sometimes people use the constant of the implementation in a test to avoid changing the unit test when changing constants.

I prefer to *not* do that because it makes it more obvious in the unit test what's going on. If the constant gets changed by accident, it gets caught by the unit test.

So this code is fine.